cypress/integration/locks.js

```diff
@@ -1,8 +1,49 @@
 describe("Locks", function() {
-    // beforeEach(function() {
-    //     cy.getUserAEurBalance().as("startingAeurBalance");
-    //     cy.getUserEthBalance().as("startingEthBalance");
-    // });
+    const getLock = (prodId, disbursedAmount) => {
+        cy.get("[data-testid=lockMenuLink]").click();
 
-    it("Should lock and release A-EUR");
+        cy.get(`[data-testid=selectLockProduct-${prodId}]`).then(() => {
+            cy
+                .get("[data-testid=lockAmountInput]")
+                .type(disbursedAmount.toString())
+                .should("have.value", disbursedAmount.toString())
+        });
+        cy.get(`[data-testid=selectLockProduct-${prodId}]`).click();
+
+        cy.get(`[data-testid=submitButton]`).click();
+        cy.get("[data-testid=EthSubmissionSuccessPanel]").contains("New Lock submitted")
+        cy.get("[data-testid=EthSubmissionSuccessPanel] >[data-testid=msgPanelOkButton]").click();
+        
+        return cy.get("[data-testid=EthConfirmationReceivedPanel]").within(() => {
+            cy.contains("New lock");
+            cy.contains("Tx hash:");
+        });
+    };
+    
+    beforeEach(function() {
+        cy.getUserAEurBalance().as("startingAeurBalance");
+        cy.getUserEthBalance().as("startingEthBalance");
+    });
+    
+    it("Should lock A-EUR", function() {
+        getLock(7, 50).then(() => {
+            cy.assertUserAEurBalanceOnUI(this.startingAeurBalance - 50);
+            cy.get("[data-testid=EthConfirmationReceivedPanel] > [data-testid=msgPanelOkButton]").click();
+        });
+    });
+
+    it("Should release locked A-EUR", function() {        
+        getLock(7, 50).then(() => {
+            cy.get("[data-testid=EthConfirmationReceivedPanel]").within(() => {
+                cy.contains("12. confirmation", { timeout: 60000 });
+            });
+            cy.get("[data-testid=myAccountMenuLink]").click();
+            cy.wait(65000).then(() => {
```

Comment:
we shouldn't use `wait(time)` in tests, it slows down tests unnecessarily (eg. running local) and it might fail unexpectedly when running on slower machines (eg. travis is having a lazy day)
we should wait for certain elements appear on UI, see loans test as example. Might worth to understand how cypress is handling this, [corresponding section in cypress docs](https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#Cypress-is-Not-Like-jQuery)